pkg/formats/format.go

```diff
@@ -25,21 +26,24 @@ type Formatter interface {
 type Format string
 
 const (
-	FORMAT_AVRO      Format = "avro"
-	FORMAT_JSON             = "json"
-	FORMAT_JSON_FLAT        = "flat_json"
-	FORMAT_NETFLOW          = "netflow"
-	FORMAT_INFLUX           = "influx"
-	FORMAT_PROM             = "prometheus"
-	FORMAT_NR               = "new_relic"
-	FORMAT_NRM              = "new_relic_metric"
-	FORMAT_SPLUNK           = "splunk"
+	FORMAT_AVRO          Format = "avro"
+	FORMAT_ELASTICSEARCH        = "elasticsearch"
+	FORMAT_JSON                 = "json"
+	FORMAT_JSON_FLAT            = "flat_json"
+	FORMAT_NETFLOW              = "netflow"
+	FORMAT_INFLUX               = "influx"
+	FORMAT_PROM                 = "prometheus"
+	FORMAT_NR                   = "new_relic"
+	FORMAT_NRM                  = "new_relic_metric"
+	FORMAT_SPLUNK               = "splunk"
 )
 
 func NewFormat(format Format, log logger.Underlying, compression kt.Compression) (Formatter, error) {
 	switch format {
 	case FORMAT_AVRO:
 		return avro.NewFormat(log, compression)
+	case FORMAT_ELASTICSEARCH:
+		return elasticsearch.NewFormat(log)
```

Comment:
Do we want / need to handle compression here also? Not sure you ever want to gzip sending into ES but its an option.